index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
```

Comment:
[ ] kindly use the header tag for your main site navigation, it's one of The HTML best practices 🤓

Reply:
If I use a header tag instead of a title tag My Portfolio is going to appear on top of the body, not on the search bar of the browser.